sidekick_core/lib/src/commands/update_command.dart

```diff
@@ -78,10 +78,19 @@ class UpdateCommand extends Command {
       pubspecKeys: ['dependencies', 'sidekick_core'],
       newMinimumVersion: to,
     );
-    _dartCommand(
-      ['pub', 'get'],
-      workingDirectory: Repository.requiredCliPackage,
-    );
+    try {
+      _dartCommand(
+        ['pub', 'get'],
+        workingDirectory: Repository.requiredCliPackage,
```

Comment:
The error isn't thrown anymore, but the output of the failed `pub get` is still printed 
```
Updating sidekick CLI sk from version 0.14.0 to 0.15.0 ...
 - Update /tool directory
 - Update entrypoint
 - Update embedded dart version
Successfully updated sidekick CLI sk from version 0.14.0 to 0.15.0!
Resolving dependencies... 
The current Dart SDK version is 2.14.0.

Because sk_sidekick requires SDK version >=2.18.7 <3.0.0, version solving failed.
```
```suggestion
        workingDirectory: Repository.requiredCliPackage,
        progress: Progress.devNull(),
```